server/debug/src/process.rs

```diff
@@ -265,3 +270,87 @@ fn ensure_no_nuls<S: AsRef<OsStr>>(s: S) -> io::Result<S> {
         Ok(s)
     }
 }
+
+
+/// Struct to capture the pid and name of a running process
+pub struct Proc {
+    pub id: u32,
+    pub name: String,
+}
+
+/// helper function to get list of running windows processes on machine
+pub fn read_win_proc() -> Vec<Proc> {
```

Comment:
Could you mark these with `#[cfg(windows)]` and `#[cfg(unix)]` and give them the same name?

It would also be good for them to return an `Iterator` rather than a `Vec`.